### PR TITLE
Replaced preview url with non-preview url

### PR DESCRIPTION
**Pull request by 96RadhikaJadhav**
# Changes
This PR will resolve issue #182 .

Replaced preview url with non-preview url in README.md.

# Submitter Checklist

These are the criteria that every PR should meet, please check them off as you
review them:

- [ ] Includes [tests](https://github.com/tektoncd/community/blob/master/standards.md#principles) (if functionality changed/added)
- [x] Commit messages follow [commit message best practices](https://github.com/tektoncd/community/blob/master/standards.md#commit-messages)

_See [the contribution guide](https://github.com/tektoncd/pipeline/blob/master/CONTRIBUTING.md) for more details._

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -5,7 +5,7 @@ share and contribute tasks and pipelines for Tekton. This repository
 contains the source code of this service.
 
 
-> Currently, Hub is launched as a preview [hub-preview.tekton.dev](https://hub-preview.tekton.dev) :mega:
+> Currently, Hub is launched as a preview [hub.tekton.dev](https://hub.tekton.dev) :mega:
```
Comment (vinamra28) [inline README.md:8]:
```suggestion
> Currently, Hub is launched as [hub.tekton.dev](https://hub.tekton.dev) :mega:
```

**Review by vinamra28 — COMMENTED** [review]
/ok-to-test

File: README.md (hunk 1 above)
Comment (96RadhikaJadhav) [reply README.md]: @vinamra28  Done.

**Comment by vinamra28** [comment]
Thanks @96RadhikaJadhav :+1: 
Can you please squash the commits as well? :slightly_smiling_face:

**Comment by vinamra28** [comment]
Thanks @96RadhikaJadhav :+1: 
Can you please squash the commits as well? :slightly_smiling_face:

**Comment by 96RadhikaJadhav** [comment]
> Thanks @96RadhikaJadhav 👍
> Can you please squash the commits as well? 🙂

@vinamra28 Done :) .

**Review by vinamra28 — COMMENTED** [review]
/lgtm
/cc @PuneetPunamiya

**Comment by PuneetPunamiya** [comment]
Thank you for the pr :call_me_hand: 
/lgtm
/approve
/meow

**Pull request merged**